src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);

```

Comment:
Don't create a new instance of Random every time the method is called. Instead, create a single static final instance of Random and reuse it.